.ci/filter_and_sign.sh

```diff
@@ -4,18 +4,35 @@ set -e
 
 proto_groups=(all les snap)
 
+# CloudFlare limits the number od DNS records per zone to 3500 for our account.
+# For this reason we need to limit the number of nodes per network (classic|mordor) and per proto (all|les|snap).
+# https://developers.cloudflare.com/dns/troubleshooting/faq/#does-cloudflare-limit-number-of-dns-records-a-domain-can-have
+limit_output_size_all=850
+limit_output_size_snap=850
+limit_output_size_les=50
+
```

Comment:
Since we know that `classic.*` has a lot, and `mordor.*` has very few, should we weight more entries to classic?

Reply:
👍  What do you think of the below values?

```
classic_limit_output_size_all=1400
classic_limit_output_size_snap=1400
classic_limit_output_size_les=50
mordor_limit_output_size_all=300
mordor_limit_output_size_snap=300
mordor_limit_output_size_les=50
```

Reply:
Yep, LGTM.